tests/functional/test_config_groups.py

```diff
@@ -448,109 +449,348 @@ def _get_config_from_group(self, cluster_group: GroupConfig):
         config_group["file"] = config_group["file"].replace("\n", "")
         return config_group
 
+    @allure.step('Check error that group config cant change')
```

Comment:
Maybe reverse `'` and `"` and use "can't"?

Reply:
changed